lib/ecl/tests/data/num_cpu5

```diff
@@ -0,0 +1,16 @@
+----------------------------------------------------------------------
+-- A very simple test case for the shared memory parallel option.
+-- --------------------------------------------------------------
+
+TITLE
+foo
+
+TITLE
+PARALLEL BENCHMARK
+
+TITLE
+foo
+
+PARALLEL
```

Comment:
Could a `-- TITLE` appended to this line (or the line below) make some havoc?

Reply:
It should not, but lets test that. I will add it to the file.

Reply:
Seems to be fine.